wallet-standard/cryptid-wallet/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "private": true,
+    "name": "cryptid-wallet-standard-wallet",
+    "author": "Identity <identity.com>",
+    "repository": "https://github.com/identity/cryptid",
+    "license": "Apache-2.0",
+    "engines": {
+        "node": ">=16"
+    },
+    "type": "module",
+    "sideEffects": false,
+    "main": "./lib/cjs/index.js",
+    "module": "./lib/esm/index.js",
+    "types": "./lib/types/index.d.ts",
+    "exports": {
+        "require": "./lib/cjs/index.js",
+        "import": "./lib/esm/index.js",
+        "types": "./lib/types/index.d.ts"
+    },
+    "scripts": {
+        "fmt": "prettier --write '{*,**/*}.{ts,tsx,js,jsx,json}'",
+        "clean": "shx mkdir -p lib && shx rm -rf lib",
+        "tsc": "tsc --build --verbose tsconfig.all.json",
+        "package": "shx echo '{ \"type\": \"commonjs\" }' > lib/cjs/package.json",
+        "build": "npm run clean && npm run tsc && npm run package"
+    },
+    "dependencies": {
+        "@solana/wallet-standard-features": "^1.0.0-rc.7",
+        "@solana/web3.js": "^1.58.0",
+        "@wallet-standard/base": "^1.0.0-rc.5",
+        "@wallet-standard/features": "^1.0.0-rc.5",
+        "@identity.com/cryptid": "0.2.0",
```

Comment:
If do a `postMessage` conversation only, we would not expect a cryptid dependency here, correct? OR do we need any library logic on this site of the adapter?